.github/workflows/release-t0rn.yml

```diff
@@ -0,0 +1,260 @@
+name: t0rn Rococo Release
+
+# Trigger only on push to this branch
+on:
+  push
+
+# TODO: uncomment after tests
+# on:
+#   push:
+#     tags:
+#       - "v[0-9]+.[0-9]+.[0-9]+-rc.[0-9]+"
+
+env:
+  PARACHAIN_NAME: t0rn
+  RUST_BACKTRACE: full
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build-release:
+    runs-on: ["self-hosted", "rust"]
+    steps:
+      - name: ☁ Checkout git repo
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+          token: ${{ secrets.GH_PAT }}
+
+      - name: ⚙️ Get nightly rust toolchain with wasm target
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly-2022-06-16
+          target: wasm32-unknown-unknown
+          components: rustfmt, clippy
+          override: true
+
+      - name: 🏭 Build circuit
+        uses: actions-rs/cargo@v1
+        continue-on-error: false
+        with:
+          command: build
+          args: --manifest-path ./node/${{ env.PARACHAIN_NAME }}-parachain/Cargo.toml --locked --release
+
+      - name: 📤 Upload collator binary
+        uses: actions/upload-artifact@v2
+        with:
+          name: ${{ github.sha }}-${{ env.PARACHAIN_NAME }}-collator-release-artifacts
+          path: ./target/release/${{ env.PARACHAIN_NAME }}-collator
+          if-no-files-found: error
+          retention-days: 1
```

Comment:
make sure this is a long time after testing too

Reply:
and why we need to store this artifact longer?
it will be added to release and will never expire then